modal/secret.py

```diff
@@ -38,4 +39,19 @@ async def _load(resolver: Resolver) -> _SecretHandle:
         super().__init__(_load, rep)
 
 
+async def _resolve_secret(resolver: Resolver, secret:_Secret) -> str:
```

Comment:
minor preference for not having this here – would rather put it in here: https://github.com/modal-labs/modal-client/blob/main/modal/object.py#L105

Reply:
To clarify: are you asking to move the method `_resolve_secret` to [object.py](https://github.com/modal-labs/modal-client/blob/main/modal/object.py#L105) (as in a simple namespace change) or to add it / integrate it to the `Handle.from_app` method?